pkg/operator/registry.go

```diff
@@ -99,6 +99,13 @@ func (m *Registry) getLatestVersionForOperator(name string) (string, error) {
 func StandardRegistry(options ...BaseOperatorOption) *Registry {
 	return &Registry{
 		operators: OperatorBuildersTree{
+			ClusterMaintenanceChangeOperatorName: map[string]OperatorBuilder{
```

Comment:
I guess this is the place where we would add new operators right ?

Reply:
Yes, it is